services/src/main/java/mops/SingleSubmitService.java

```diff
@@ -0,0 +1,39 @@
+package mops;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import mops.rollen.Student;
+@SuppressWarnings("PMD.DataflowAnomalyAnalysis")
+public class SingleSubmitService {
+
+  public void addStudentAsSubmitted(Fragebogen fragebogen, Student student) {
+    fragebogen.addStudentAsSubmitted(student);
+  }
+
+  public boolean hasSubmitted(Fragebogen fragebogen, Student student) {
+    List<Student> abgegebeneStudierende = fragebogen.getAbgegebeneStudierende();
+    for (Student studentin : abgegebeneStudierende) {
+      if (studentin.equals(student)) {
+        return true;
+      }
+    }
+    return false;
+  }
```

Comment:
Here its better to just use contains I think.
return fragebogen.getAbgegebeneStudierende().contains(student);
Not sure if  this needs to be in a Service to be honest.